src/applications/pact-act/utilities/page-navigation.js

```diff
@@ -9,7 +9,7 @@ import {
 } from './shared';
 
 /** ================================================================
- * Responsible for determining next question in flow, or redirecting to a results screen
+ * Responsible for determining next question in flow, or redirecting to a results page
```

Comment:
Noticed inconsistent usage of "results page" vs. "results screen" so I changed it all to "page"